tests/run/quote-toExprOfTuple/Macro_1.scala

```diff
@@ -0,0 +1,14 @@
+import scala.quoted._
+
+object Macro {
```

Comment:
This test should be in `tests/run-macros`

Reply:
That fixed it. It was probably a bad interaction with the non bootstrapped compiler.